qiita_db/test/test_processing_job.py

```diff
@@ -873,28 +876,28 @@ def _set_allocation(memory):
         #                                         27*1000 ~ 27000
         _set_allocation('{samples}*1000')
         self.assertEqual(
-            job_not_changed.get_resource_allocation_info(),
-            '-p qiita -N 1 -n 5 --mem 120gb --time 130:00:00')
-        self.assertEqual(job_changed.get_resource_allocation_info(),
-                         '-p qiita --mem 26K')
+            job_not_changed.resource_allocation_info,
+            '-p qiita -N 1 -n 5 --mem 120gb --time 130:00:00 --nice 10000')
+        self.assertEqual(job_changed.resource_allocation_info,
+                         '-p qiita --mem 26K --nice 10000')
 
         # a little more complex ((samples+columns)*1000000)+4000000
         #                       ((   27  +  31   )*1000000)+4000000 ~ 62000000
         _set_allocation('(({samples}+{columns})*1000000)+4000000')
         self.assertEqual(
-            job_not_changed.get_resource_allocation_info(),
-            '-p qiita -N 1 -n 5 --mem 120gb --time 130:00:00')
-        self.assertEqual(job_changed.get_resource_allocation_info(),
-                         '-p qiita --mem 80M')
+            job_not_changed.resource_allocation_info,
+            '-p qiita -N 1 -n 5 --mem 120gb --time 130:00:00 --nice 10000')
+        self.assertEqual(job_changed.resource_allocation_info,
+                         '-p qiita --mem 80M --nice 10000')
 
         # now something real input_size+(2*1e+9)
         #                        116   +(2*1e+9) ~ 2000000116
         _set_allocation('{input_size}+(2*1e+9)')
         self.assertEqual(
-            job_not_changed.get_resource_allocation_info(),
-            '-p qiita -N 1 -n 5 --mem 120gb --time 130:00:00')
-        self.assertEqual(job_changed.get_resource_allocation_info(),
-                         '-p qiita --mem 2G')
+            job_not_changed.resource_allocation_info,
+            '-p qiita -N 1 -n 5 --mem 120gb --time 130:00:00 --nice 10000')
+        self.assertEqual(job_changed.resource_allocation_info,
+                         '-p qiita --mem 2G --nice 10000')
 
         # restore allocation
```

Comment:
I haven't seen a test that confirms a return of niceness level 5000 for admins. Is this an easy test to add? It might be good to know that niceness can be something other than 10000.

Reply:
See https://github.com/qiita-spots/qiita/pull/3330/files#diff-99af21f52020eab43de2a74b4ddeb3b48475c35fcc8c07e3a08cef55e8cc3c61R519